build_tools/github_actions/lint_clang_format.sh

```diff
@@ -51,7 +49,12 @@ fi
 echo "Running clang-format [mode=$FORMAT_MODE]..."
 echo "  Files: " "${CHANGED_FILES[@]}"
 if [[ $FORMAT_MODE == 'fix' ]]; then
-  clang-format --style=google -i "${CHANGED_FILES[@]}"
+  clang-format -i "${CHANGED_FILES[@]}"
 else
-  clang-format --style=google --dry-run --Werror "${CHANGED_FILES[@]}"
+  clang-format --dry-run --Werror "${CHANGED_FILES[@]}" || {
+      echo "Please format your code before pushing:"
+      echo "  ./build_tools/github_actions/lint_clang_format.sh -f"
+      echo "Make sure you are using the right major version of clang-format:"
+      echo "  $(clang-format --version)"
+    }
```

Comment:
Can you try making this fail? I'm curious if `echo` being the last command will make it seem like the job actually passed in GH Actions.

Reply:
It absolutely will! Thanks, good catch.